src/BigBlueButton.php

```diff
@@ -133,10 +113,27 @@ public function __construct(?string $baseUrl = null, ?string $secret = null, ?Tr
         }
 
         // Keeping backward compatibility with older deployed versions
-        $this->securitySecret = $secret ?: getenv('BBB_SECURITY_SALT') ?: getenv('BBB_SECRET');
-        $this->bbbServerBaseUrl = $baseUrl ?: getenv('BBB_SERVER_BASE_URL');
+        $securitySecret = $secret ?: getenv('BBB_SECURITY_SALT') ?: getenv('BBB_SECRET');
+
+        if (false === $securitySecret) {
+            // @codeCoverageIgnoreStart
+            @trigger_error(\sprintf('Constructing "%s" without passing a secret is deprecated since 6.0 and will throw an exception in 7.0.', self::class), \E_USER_DEPRECATED);
+            $this->securitySecret = ''; // previous behaviour
+        // @codeCoverageIgnoreEnd
+        } else {
+            $this->securitySecret = $securitySecret;
+        }
 
-        $this->hashingAlgorithm = $hashingAlgorithm;
+        $bbbServerBaseUrl = $baseUrl ?: getenv('BBB_SERVER_BASE_URL');
+
+        if (false === $bbbServerBaseUrl) {
+            // @codeCoverageIgnoreStart
+            @trigger_error(\sprintf('Constructing "%s" without passing a server base URL is deprecated since 6.0 and will throw an exception 7.0.', self::class), \E_USER_DEPRECATED);
```

Comment:
```suggestion
            @trigger_error(\sprintf('Constructing "%s" without passing a server base URL is deprecated since 6.0 and will throw an exception in 7.0.', self::class), \E_USER_DEPRECATED);
```